papaparse.js

```diff
@@ -850,9 +850,9 @@
 				{
 					_results.data[i] = row;
 					if (j > _fields.length)
-						addError("FieldMismatch", "TooManyFields", "Too many fields: expected " + _fields.length + " fields but parsed " + j, i);
+						addError("FieldMismatch", "TooManyFields", "Too many fields: expected " + _fields.length + " fields but parsed " + j, self.streamer._rowCount + i);
 					else if (j < _fields.length)
-						addError("FieldMismatch", "TooFewFields", "Too few fields: expected " + _fields.length + " fields but parsed " + j, i);
+						addError("FieldMismatch", "TooFewFields", "Too few fields: expected " + _fields.length + " fields but parsed " + j, self.streamer._rowCount + i);
```

Comment:
You're right, it's possible to add this in addError... like this:

`row: row ? self.streamer._rowCount + row : row`

"MissingQuotes" is a little harder to fix. I can see an option in handing _rowCount to the _parser (like: `_parser.rowCount = self.streamer._rowCount;`) but this would probably end up in some core parser tests failing, as the variable cannot get handed in there working directly on the core parser. As long a you don't test for it not being defined like: `row: (this.rowCount ? this.rowCount : 0) + data.length,	// row has yet to be inserted`
Seems a little off to me.